charts/karpenter/templates/configmap-logging.yaml

```diff
@@ -25,14 +25,14 @@ data:
       "errorOutputPaths": [{{ include "karpenter.controller.errorOutputPathsList" . }}],
       "encoding": "{{ .Values.logEncoding }}",
       "encoderConfig": {
-        "timeKey": "time",
-        "levelKey": "level",
-        "nameKey": "logger",
-        "callerKey": "caller",
-        "messageKey": "message",
-        "stacktraceKey": "stacktrace",
-        "levelEncoder": "capital",
-        "timeEncoder": "iso8601"
+        "timeKey": "{{ .Values.encoderConfig.timeKey }}",
```

Comment:
Possible to do a merge of arbitrary properties, to avoid specifying these one by one?